src/app/resources/resources.component.html

```diff
@@ -1,58 +1,104 @@
-<div>
-  <button mat-raised-button routerLink="add" color="primary" i18n>Add Resource</button>
-</div>
-<div>
-  <mat-form-field>
-    <input matInput (keyup)="applyResFilter($event.target.value)" i18n-placeholder placeholder="Search">
-  </mat-form-field>
-</div>
-<mat-table #table [dataSource]="resources">
-  <ng-container matColumnDef="title">
-    <mat-header-cell *matHeaderCellDef></mat-header-cell>
-    <mat-cell *matCellDef="let element" class="list-item-info">
-      <h3>{{element.title}}</h3>
-      <div class="list-item-content">
-        Author: {{ element.author }},Year: {{ element.year }}, Media: {{ element.Medium }}, Language: {{ element.language }}, Subject/s: {{element.subject}}, Publisher/Attribution: {{ element.Publisher }}, Link to License: {{element.linkToLicense}}, Resource: {{ element.resourceFor }}, Resource Type: {{ element.resourceType }}
-      </div>
-      <div class="list-item-content">
-        <a mat-raised-button color="primary" i18n [routerLink]="['/resources/view', element._id]">View Details</a>
-        <a mat-raised-button color="primary" i18n [routerLink]="['/resources/wip']">Add To Library</a>
-        <a mat-raised-button color="primary" i18n [routerLink]="['/resources/wip']">Feedback</a>
-        <button mat-raised-button color="warn" (click)="deleteClick(element)" i18n>Delete</button>
-      </div>
-    </mat-cell>
-  </ng-container>
-  <ng-container matColumnDef="rating">
-    <mat-header-cell *matHeaderCellDef></mat-header-cell>
-    <mat-cell *matCellDef="let element">
-      <div class="list-item-rating">
-        <span class="stars">
-          <mat-icon>star</mat-icon><mat-icon>star</mat-icon><mat-icon>star</mat-icon><mat-icon>star</mat-icon><mat-icon>star</mat-icon>
-          <!-- used simple arithmetic to get 20 and converted into % -->
-          <span [ngStyle]="{'width': getRating(element.sum, element.timesRated)}">
+<!--Used to approximate design.  Should be replaced before merging"-->
+<div class="background"></div>
+
+<mat-toolbar>
+  <mat-toolbar-row>
+    <button mat-icon-button (click)="goBack()"><mat-icon>arrow_back</mat-icon></button>
+    <span i18n>Library</span>
+    <span class="toolbar-fill"></span>
+    <mat-icon>search</mat-icon>
+    <mat-form-field class="font-size-1">
+      <input matInput (keyup)="applyResFilter($event.target.value)" i18n-placeholder placeholder="Type text to search...">
+    </mat-form-field>
+  </mat-toolbar-row>
+</mat-toolbar>
+
+<div class="space-container">
+  <mat-toolbar>
+    <mat-toolbar-row class="primary-color font-size-1">
+      <mat-checkbox (change)="$event ? masterToggle() : null"
+        [checked]="selection.hasValue() && isAllSelected()"
```

Comment:
This seems redundant.  Why is there the check on `selection.hasValue` in addition to `isAllSelected`?